src/Avalonia.Controls/ColumnDefinition.cs

```diff
@@ -108,5 +109,32 @@ public GridLength Width
         internal override GridLength UserSizeValueCache => this.Width;
         internal override double UserMinSizeValueCache => this.MinWidth;
         internal override double UserMaxSizeValueCache => this.MaxWidth;
+
+        /// <inheritdoc/>
+        public bool Equals(ColumnDefinition other)
+        {
+            return other != null
+                ? Hash(this) == Hash(other)
```

Comment:
`Equals` is still comparing hashes here.

Reply:
sorry, had problems with my git config...